utils/colors.ts

```diff
@@ -1,4 +1,13 @@
-const colors = {
+interface Colors{
+	"primary":string,
+	"secondary":string,
+	"text":string,
+	"invert":string,
```

Comment:
just noticed it, but invert is not a name of a color... and the value you pass it is not a color either.
if it's not all colors maybe it should be `themeCSSVariables`? I dunno...

Reply:
noted